textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/queryrewriter/WordBase.java

```diff
@@ -0,0 +1,35 @@
+package edu.uci.ics.textdb.dataflow.queryrewriter;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.HashSet;
+
+/**
+ * Created by shiladityasen on 4/30/16.
+ * Class to provide simple abstraction of a dictionary of words with constant retrieval time
+ * Used by FuzzyTokenizer to process query
+ */
+public class WordBase
+{
+
+    private HashSet<String> wordBase;
+
+    public WordBase(String wordBaseSourceFileName) throws IOException {
+        wordBase = new HashSet<String>();
+        String line;
+
+        URL wordBaseURL = getClass().getResource("/queryrewriter/"+wordBaseSourceFileName);
```

Comment:
@kishore-narendran .. Actually it is better to include the whole path `"/queryrewriter/"+wordBaseSourceFileName` as parameter to the WordBase constructor.